monitoring/port.go

```diff
@@ -103,6 +104,14 @@ func (c *portChecker) checkProcess(proc process, reporter health.Reporter) bool
 		if r.Protocol != proc.socket.proto() {
 			continue
 		}
+		// if prechecks have been run multiple times quickly, sockets from
+		// the ping-pong test may still be lingering in time-wait state and
+		// should eventually go away
```

Comment:
Nit: I would maybe just preface this with `ignore time-wait state`... explanation of why it's being ignored. It might be just me, but for some reason I have a hard time parsing this as is.